cmd/create.go

```diff
@@ -313,7 +326,23 @@ func createHost(provider, name, region, zone, projectID, userData, inletsPort st
 			UserData: base64.StdEncoding.EncodeToString([]byte(userData)),
 			Additional: map[string]string{
 				"inlets-port": inletsPort,
-				"pro": fmt.Sprint(pro),
+				"pro":         fmt.Sprint(pro),
+			},
+		}, nil
+	} else if provider == "azure" {
+		return &provision.BasicHost{
+			Name:     name,
+			OS:       "Additional.imageOffer",
+			Plan:     "Standard_B1ls",
```

Comment:
Is this the cheapest option with an IP?

Reply:
Yes it is the cheapest plan, with 1 core, 0.5 GiB RAM, ~$4 / month (depending on region). see https://azureprice.net/ and https://azure.microsoft.com/en-us/pricing/details/virtual-machines/linux/